yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -1291,6 +1352,95 @@ module vyatta-protocols-frr-bgp-v1 {
 						}
 						uses neigh_peer_grp_ipv4_lists;
 					}
+
+          container ipv4-vpn {
```

Comment:
This container is under the peer-group configuration rather than the neighbor configuration, but the command translations above are looking for neighbor configuration.

So I'm not sure these diffs are working right now? If you want per-neighbor configuration then I think this should be under the bgp-params-neighbor grouping.

Reply:
Sorry, actually the container ipv4-vpn is placed under the bgp-params-neighbor grouping but somehow it went in wrong place in this PR.
It is fixed now, will send it for review shortly. 
I have attached a screenshot verifying that. 
![Saving-FRR_Config_file](https://user-images.githubusercontent.com/62253226/79356131-3324f400-7f5c-11ea-8513-4a206af9d54c.png)